tests/integration/api/test_target_correctness.py

```diff
@@ -206,3 +207,11 @@ def test_forward_aggregate(
             expected_values["POINT_IN_TIME"], utc=True
         ).dt.tz_localize(None)
         fb_assert_frame_equal(df_targets, expected_values)
+
+        # Build full materialized Target observation table
```

Comment:
Would be nice if there is a test to cover the intended workflow: materialise a target, then use the resulting observation table to materialise some features.

Reply:
i'll add in a follow-up!